packages/e2e/src/helpers/contacts.helper.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+import ContactsPage from "../page-objects/contacts.page"
+import NavigationTabs from "../page-objects/tabs.page"
+import ModalContactsPage from "../page-objects/modal-contacts.page"
+import { waitForClickableAndClick } from "../helpers/general.helper"
+/** Add new contact */
+export const addNewContact = async (
+  firstNameText: string,
+  lastNameText: string,
+  primaryPhoneText: string,
+  seondaryPhoneText?: string,
```

Comment:
typo in secondary :)

Reply:
https://github.com/mudita/mudita-center/pull/1160/commits/fd4cd48c36293960964f1aedf25688ac27187039